app/src/Routes/Works/Works.js

```diff
@@ -1,40 +1,125 @@
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import List from '@material-ui/core/List';
-import ListItem from '@material-ui/core/ListItem';
+import Link from '@material-ui/core/Link';
 import OpenInNewIcon from '@material-ui/icons/OpenInNew';
 import Typography from '@material-ui/core/Typography';
 import * as works from './works.json';
 import css from './Works.module.scss';
 
 const Piece = (piece) => {
-  const { title, type, instruments, detailLink, external } = piece;
+  const {
+    title,
+    type,
+    instruments,
+    duration,
+    description,
+    publisher,
+    movements,
+    attribution,
+    composerNotes,
+    detailLink,
+    external
+  } = piece;
   const key = `${title}-${type}`;
   const allInstruments = instruments && instruments.join(', ');
+  const allMovements = movements && movements.join(', ');
+  const allComposerNotes = composerNotes && composerNotes.join(', ');
+  const hasMoreInfo = !!allInstruments || !!allMovements || !!allComposerNotes;
+
+  console.log('piece', piece);
 
   return (
-    <ListItem
-      key={key}
-      href={detailLink}
-      target={external ? '_blank' : ''}
-      title={`View more details about ${title}`}
-      component="a"
-      className={css.piece}
-      button
-    >
-      <div className={css.title}>{title}</div>
-      {type && <div className={css.type}>({type})</div>}
-      {allInstruments && (
-        <div className={css.instruments}>Instruments: {allInstruments}</div>
-      )}
-      {external && (
-        <OpenInNewIcon
-          className={css.openInNewIcon}
-          color="disabled"
-          fontSize="small"
-          titleAccess="Opens in new window"
-        />
-      )}
-    </ListItem>
+    <Accordion key={key}>
+      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+        <div>
+          <div className={css.piece}>
+            <div className={css.title}>
+              <Link
+                href={detailLink}
+                target={external ? '_blank' : ''}
+                title={`View more details about ${title}`}
+                className={css.detailLink}
+                onClick={(e) => e.stopPropagation()}
+                color="textPrimary"
+              >
+                <span>{title}</span>
+                {external && (
+                  <OpenInNewIcon
+                    className={css.openInNewIcon}
+                    color="disabled"
+                    fontSize="small"
+                    titleAccess="Opens in new window"
+                  />
+                )}
+              </Link>
+            </div>
+            {type && <div className={css.type}>({type})</div>}
+          </div>
+          <Typography variant="caption" color="textSecondary">
+            {publisher && `Publisher: ${publisher}`}
+            {publisher && duration && ` | `}
+            {duration && `Duration: ${duration}`}
+          </Typography>
+        </div>
+      </AccordionSummary>
+      <AccordionDetails>
```

Comment:
accordion is a great idea, keeps the routing simple and works on mobile.

is there a way to start with it open if there's an anchor link?

Reply:
good call